vignettes/wip/dm-db.Rmd

```diff
@@ -107,11 +146,9 @@ my_dm_trans <-
 my_dm_trans
 ``````
 
-- Copying tables into existing objects: `copy_to()` demonstrated further below, in "deploying" section
-
-## Transient nature of operations
+And, like {dbplyr}, results are never written to a database unless explicitly requested.
 
-- Like dbplyr, results are never written to database unless asked explicitly
+#ASK KIRILL - what exactly is happening here
```

Comment:
"flatten" is an operation that starts with a table and brings in all immediately related tables with a join. The operation returns a (lazy) table. The `sql_render()` prints the SQL generated for that operation.

We don't really need to show `sql_render()` if it complicates things. The "lazy table" nature is visible in the output too.

Does that help?